spring-cloud-aws-samples/spring-cloud-aws-parameter-store-sample/src/main/java/io/awspring/cloud/parameterstore/sample/SpringCloudAwsParameterStoreSample.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2013-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.awspring.cloud.parameterstore.sample;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+
+@SpringBootApplication
+public class SpringCloudAwsParameterStoreSample {
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(SpringCloudAwsParameterStoreSample.class);
+
+	public static void main(String[] args) {
+		SpringApplication.run(SpringCloudAwsParameterStoreSample.class, args);
+	}
+
+	@Bean
+	ApplicationRunner applicationRunner(@Value("${.message}") String message) {
```

Comment:
Should it be `${.message}` or `${message}`?

Reply:
Since config path is not ending with "/" it parameter store integration will put it as ".message"
aws-parameterstore:/config/spring
This is current implementation of parameter store in 2.3.0.
If we end config with "/"  parameter store integration will put it as "message".
Will end path with "/" and will replace it with `(@Value("${.message}"`.